src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.eligibility.unit.spec.jsx

```diff
@@ -355,7 +376,7 @@ describe.skip('VAOS <VAFacilityPage> eligibility check', () => {
 
       await screen.findByText(/Select a VA facility/i);
 
-      fireEvent.click(await screen.findByLabelText(/Fake facility name 5/i));
+      fireEvent.click(await screen.findByLabelText(/Fake facility name 1/i));
       fireEvent.click(screen.getByText(/Continue/));
       await screen.findByTestId('eligibilityModal');
```

Comment:
suggest adding the no clinic message
```suggestion
      fireEvent.click(await screen.findByLabelText(/Fake facility name 1/i));
      fireEvent.click(screen.getByText(/Continue/));
      await screen.findByTestId('eligibilityModal');
      expect(screen.getByRole('alertdialog')).to.be.ok;
      fireEvent.click(screen.getByRole('button', { name: /Continue/i }));

      expect(
        await screen.findByText(
          /This facility doesn’t have any available clinics/i,
        ),
      ).to.be.ok;

```

Reply:
Done.